### PR TITLE
Implement the F option in RedrawAxis to redraw the plots frame

### PR DESCRIPTION
**Pull request by couet**
There was the need to redraw the frame plot in some cases.
This was requested here:
https://root-forum.cern.ch/t/change-background-color-only-in-a-region-of-a-th1

### CODE REVIEW

File: graf2d/gpad/src/TPad.cxx [hunk 1]
```diff
@@ -5309,6 +5312,16 @@ void TPad::RedrawAxis(Option_t *option)
       else                   hobj->DrawCopy("sameaxis");
    }
 
+   if (opt.Contains("f")) {
+      auto b = new TBox(gPad->GetUxmin(), gPad->GetUymin(),
+                        gPad->GetUxmax(), gPad->GetUymax());
+      b->SetFillStyle(0);
+      b->SetLineStyle(1);
```
Comment (bellenot) [inline graf2d/gpad/src/TPad.cxx:5319]: So the fill and line styles are hard-coded?

Comment (couet) [reply graf2d/gpad/src/TPad.cxx:5319]: Yes, it is to redraw the line frame around a plot when some drawing just at the edge erased it (to be used in seldom cases but there was a request and it makes sense).  RedrawAxis is the place to put that. What we want to keep are the Line Width and the Line Color only.

Comment (couet) [reply graf2d/gpad/src/TPad.cxx:5319]: But yes in principal we cans that this two values from the frame ... let me try

Comment (couet) [reply graf2d/gpad/src/TPad.cxx:5319]: Well only the line style .. the fill style must be hollow (to redraw the line border only)

Comment (bellenot) [reply graf2d/gpad/src/TPad.cxx]: OK, you know better than me...

**Pull request merged**